README.md

```diff
@@ -94,32 +94,32 @@ The passphrase argument is optional if `STATICRYPT_PASSWORD` is set in the envir
           --remember-label          Label to use for the "Remember me" checkbox.
                                                    [string] [default: "Remember me"]
       -s, --salt                    Set the salt manually. It should be set if you
-                                    want use "Remember me" through multiple pages.
-                                    It needs to be a 32 character long hexadecimal
+                                want to use "Remember me" through multiple pages.
+                                    It needs to be a 32-character-long hexadecimal
                                     string.
                                     Include the empty flag to generate a random salt
                                     you can use: "statycrypt -s".           [string]
           --share                   Get a link containing your hashed password that
                                     will auto-decrypt the page. Pass your URL as a
                                     value to append "?staticrypt_pwd=<hashed_pwd>",
-                                    or leave empty to display the hash to append.
+                                    or leave it empty to display the hash to append.
                                                                             [string]
-      -t, --title                   Title for output HTML page.
+      -t, --title                   Title for an output HTML page.
                                                 [string] [default: "Protected Page"]
 
 ## HOW STATICRYPT WORKS
 
 So, how can you password protect html without a back-end?
 
-StatiCrypt uses the [crypto-js](https://github.com/brix/crypto-js) library to generate a static, password protected page that can be decrypted in-browser. You can then just send or upload the generated page to a place serving static content (github pages, for example) and you're done: the page will prompt users for password, and the javascript will decrypt and load your HTML, all done in browser.
+StatiCrypt uses the [crypto-js](https://github.com/brix/crypto-js) library to generate a static, password protected page that can be decrypted in-browser. You can then just send or upload the generated page to a place serving static content (github pages, for example) and you're done: the page will prompt users for a password, and the javascript will decrypt and load your HTML, all done in a browser.
```

Comment:
Super tiny detail - I'd say all done in `the browser` instead of `a browser`, to me it underlines that it happens client side